meta/meta.go

```diff
@@ -14,11 +14,17 @@ import (
 )
 
 // The main version number that is being run at the moment.
-var SDKVersion = "2.10.1"
+//
+// Deprecated: Use Go standard library [runtime/debug] package build information
+// instead.
+var SDKVersion = "2.30.0"
```

Comment:
super nit: should this be `2.29.0`, then will be updated by the release when it runs

Reply:
Yep, let's do that.

```suggestion
var SDKVersion = "2.29.0"
```